spec/directives/dateInput.directive.spec.js

```diff
@@ -86,4 +86,22 @@ describe('directive: pa-date-input', function() {
 
   });
 
+  describe('test initialize from string', function() {
```

Comment:
We should choose a better title for this describe block. It does not clearly states what the block tests. Maybe something like 'when the model is a string'.